tests/components/dsmr/test_sensor.py

```diff
@@ -107,6 +152,10 @@ async def test_default_setup(hass, mock_connection_factory):
     assert gas_consumption.state == "745.695"
     assert gas_consumption.attributes.get("unit_of_measurement") == VOLUME_CUBIC_METERS
 
+    await hass.config_entries.async_unload(mock_entry.entry_id)
+
+    assert mock_entry.state == "not_loaded"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6103615)